package.json

```diff
@@ -144,11 +144,11 @@
     "@angular-eslint/eslint-plugin-template": "^13.0.1",
     "@angular-eslint/schematics": "^13.0.1",
     "@angular-eslint/template-parser": "^13.0.1",
-    "@angular/cli": "^13.1.2",
+    "@angular/cli": "^13.2.0-rc.1",
     "@angular/compiler-cli": "^13.1.1",
     "@angular/language-service": "^13.1.1",
     "@ngrx/store-devtools": "^13.0.2",
-    "@nguniversal/builders": "^13.0.0",
+    "@nguniversal/builders": "^13.1.0-next.1",
     "@schematics/angular": "^13.0.1",
```

Comment:
1. Let's bump them all, to align versions.

2. Please also bump those guys (I couldn't add a comment for those lines this PR):
```
    "@angular-builders/custom-webpack": "^13.0.0",
    "@angular-devkit/build-angular": "^13.1.2",
    "@angular-devkit/core": "^13.1.2",
``` 

3. Let's also bump the "peerDependencies" in all libraries' package.jsons, e.g. (`projects/core/package.json`)

4. In the end, let's align the versions in the `dependencies.json`

Reply:
`@nguniversal/builders` haven't received any release for 12 days, so we're up to date here
`@angular-builders/custom-webpack` also haven't received a stable release, only 13.1.0-beta available